google/cloud/forseti/services/scanner/dao.py

```diff
@@ -14,25 +14,24 @@
 
 """ Database access objects for Forseti Scanner. """
 
-from collections import defaultdict
 import hashlib
 import json
+from collections import defaultdict
 
-from sqlalchemy import and_
+from google.cloud.forseti.common.data_access import violation_map as vm
```

Comment:
Why is the import order changed here?  
```
Imports should be grouped with the order being most generic to least generic:
- standard library
- third-party library
- application library
```
https://github.com/google/styleguide/blob/gh-pages/pyguide.md

Reply:
I must have accidentally used the "Optimize imports" feature in PyCharm, thanks for catching this!